src/register-org/components/check-your-answers/check-your-answers.component.html

```diff
@@ -0,0 +1,194 @@
+              [routerLink]="'/register-org-new/registered-with-regulator'">
+              Change<span class="govuk-visually-hidden">What regulator are you(as an individual) registered with?</span>
+            </a>
+          </dd>
+        </div>
+      </dl>
+      <br>
+      <h2 class="govuk-heading-m">Administration details</h2>
+
+      <dl class="govuk-summary-list govuk-!-margin-bottom-0">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">First name</dt>
+          <dd class="govuk-summary-list__value">{{ registrationData.contactDetails.firstName }}</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="javascript:void(0)" [routerLink]="'/register-org-new/contact-details'">
+              Change<span class="govuk-visually-hidden">First name</span>
+            </a>
+          </dd>
+        </div>
+      </dl>
+
+      <dl class="govuk-summary-list govuk-!-margin-bottom-0">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Last name</dt>
+          <dd class="govuk-summary-list__value">{{ registrationData.contactDetails.lastName }}</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="javascript:void(0)" [routerLink]="'/register-org-new/contact-details'">
+              Change<span class="govuk-visually-hidden">Last name</span>
+            </a>
+          </dd>
+        </div>
+      </dl>
+
+      <dl class="govuk-summary-list govuk-!-margin-bottom-0">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Email address</dt>
+          <dd class="govuk-summary-list__value">{{ registrationData.contactDetails.workEmailAddress }}</dd>
+          <dd class="govuk-summary-list__actions">
+            <a class="govuk-link" href="javascript:void(0)" [routerLink]="'/register-org-new/contact-details'">
+              Change<span class="govuk-visually-hidden">Email address</span>
+            </a>
+          </dd>
+        </div>
+      </dl>
+      <br>
+      <h2 class="govuk-heading-m">Confirm your registration details</h2>
+      <p>Ensure that:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>your personal and organisation detils are correct</li>
+        <li>you have used your registered office address and not a personal or home address</li>
+        <li>you are creating an account for your organisation for the first time</li>
+        <li>you are authorised to register an account on behalf of your ogrnaisation</li>
+      </ul>
+
+      <div class="govuk-warning-text">
+        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+        <strong class="govuk-warning-text__text">
+          <span class="govuk-warning-text__assistive">Warning</span>
+          Make sure all information is correct as your appliction will be rejected if it contains errors.
```

Comment:
@DavidJayakumar - typo here; should be `application`

Reply:
done